Devninos

Ethan-Sarricolea opened the pull request:
Tengo un error en el dao, pero no se exactamente a que se debe, en cuanto a lo demas espero que este mejor.

Ethan-Sarricolea committed:
Add child.dao, child.types and father module with get and post endpoints

i have not yet test to check the operation

Ethan-Sarricolea committed:
index fusion

i resolved the  merge conflict  error functioning both codes

Ethan-Sarricolea committed:
Correcciones de factory e intento de correccion de uso de la base de datos

Me da un error en el dao de child, pero no estoy del todo seguro por que

rafabelts reviewed (changes requested):
Notas:
- Pq haces un factory para la db?
- El dao porque no se agrega en el archivo ya existente
- Lo mismo con el service

Ethan-Sarricolea committed:
Correcciones de dao y services

Corregí el dao y el services, tambien elimine el database.factory y agregue un notNull al schema en fatherId

Ethan-Sarricolea committed:
Eliminando comentarios innecesarios

Ethan-Sarricolea committed:
main merge

Ethan-Sarricolea committed:
Correcciones en child.dao, father services y controller

rafabelts committed:
added create endpoint for childs

rafabelts reviewed (approved):
LGTM
